examples/flutter_gallery/lib/gallery/demos.dart

```diff
@@ -558,19 +558,16 @@ List<GalleryDemo> _buildGalleryDemos() {
 
   // Keep Pesto around for its regression test value. It is not included
   // in (release builds) the performance tests.
```

Comment:
I think you'll want to get rid of this comment too...

Reply:
Done. Thanks!